master/docs/developer/www.rst

```diff
@@ -617,6 +617,139 @@ Several methods are added to each "restangularized" objects, aside from get(), p
     Call the control data api.
     This builds up a POST with jsonapi encoded parameters
 
+DataService
+.............
+
+DataService is the future replacement of BuildbotService for accessing the Buildbot data API.
```

Comment:
I am currently rewriting the base and the plugins to the DataService, but I wanted this to be correct. So this is independent from that and I can remove the 'future' in that PR.